MCMC/utils.py

```diff
@@ -2,6 +2,33 @@
 import numpy as np
 from numba import jit
 
+def avg_nn(sim_obj, n_frames, r_max=3):
+    box = [sim_obj.L, sim_obj.L,0]
+    frame_avg_nn = []
+    for points in sim_obj.system_history[-n_frames:]:
+        points = np.append(
+            points, np.zeros((points.shape[0], 1)),axis=1
+        )
+        aq = freud.locality.AABBQuery(box, points)
+        nlist = aq.query(points, {'r_max': 3}).toNeighborList()
+        neighbors = []
```

Comment:
What does the aq function/variable do?

Reply:
This is a function in the freud analysis package we use a lot. Here it is essentially being used to search through the particles in the system (points and box) to find neighbors, or neighborhoods of particles.  The docs for freud have a pretty good overview of it [here](https://freud.readthedocs.io/en/latest/gettingstarted/tutorial/neighborfinding.html#neighbors)